nextcord/application_command.py

```diff
@@ -96,7 +96,8 @@ def to_register(self) -> List[ApplicationCommand]:
         return self.__cog_to_register__
 
 
-class SlashOption:
+# Extends Any so that type checkers won't complain that it's a default for a parameter of a different type
+class SlashOption(Any):
```

Comment:
This is breaking autodocs for some reason. We need to either find an alternative workaround for typing, or a way to have autodocs ignore this.